les/handler.go

```diff
@@ -281,6 +286,19 @@ func (pm *ProtocolManager) handle(p *peer) error {
 		p.Log().Debug("Light Ethereum handshake failed", "err", err)
 		return err
 	}
+
+	if !pm.lightSync && !p.Peer.Info().Network.Trusted {
+		addr, ok := p.RemoteAddr().(*net.TCPAddr)
```

Comment:
Note: `net.Addr` supports `String`, it's one of the methods, so you don't need to check for `TCPAddr` here.

Reply:
That includes port too. I just want to use the IP address here because it is very easy for the client to change port.